pkgs/applications/misc/procmail/default.nix

```diff
@@ -1,22 +1,22 @@
-{ lib, stdenv, fetchurl }:
+{ lib, stdenv, fetchurl, fetchFromGitHub, writeText, bash,
+  msmtp }:
 
 stdenv.mkDerivation rec {
   pname = "procmail";
-  version = "3.22";
+  version = "3.24";
 
-  src = fetchurl {
-    url = "ftp://ftp.fu-berlin.de/pub/unix/mail/procmail/procmail-${version}.tar.gz";
-    sha256 = "05z1c803n5cppkcq99vkyd5myff904lf9sdgynfqngfk9nrpaz08";
+  sendmail = msmtp;
+
+  src = fetchFromGitHub {
+    owner = "BuGlessRB";
+    repo = "procmail";
+    rev = "v${version}";
+    sha256 = "sha256-z0g+IeSldUSwv/W4pAQhtWiughujxsEPgDcsW9Fx3jY=";
   };
 
-  patches = [
-    ./CVE-2014-3618.patch
-    (fetchurl {
-      url = "https://sources.debian.org/data/main/p/procmail/3.22-26/debian/patches/30";
-      sha256 = "11zmz1bj0v9pay3ldmyyg7473b80h89gycrhndsgg9q50yhcqaaq";
-      name = "CVE-2017-16844";
-    })
-  ];
+  buildInputs = [ sendmail ];
+
+  patches = [ ./autoconf.patch ];
```

Comment:
Please move patches back to where it was. Also the name could be more descriptive.